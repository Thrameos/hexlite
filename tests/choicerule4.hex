d(1). d(2). d(3).

1 <= { p(X) } :- d(X).
