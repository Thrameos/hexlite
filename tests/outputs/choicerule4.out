{d(1),d(2),d(3),p(1),p(2),p(3)}
